src/virtualenv/create/creator.py

```diff
@@ -90,6 +92,13 @@ def add_parser_arguments(cls, parser, interpreter, meta, app_data):
             help="remove the destination directory if exist before starting (will overwrite files otherwise)",
             default=False,
         )
+        parser.add_argument(
+            "--no-vcs-ignore",
+            dest="vcs_ignore",
```

Comment:
I don't like how you negated the flag name, but kept the destination positive. Please keep them in sync. This flag should be ``--no-vcs-ignore``, destination `no_vcs_ignore` and should `store_true` with default `False`.

Reply:
Done.  Wasn't really any reason for old way, other than the double negative on [line 172](https://github.com/pypa/virtualenv/pull/2004/files#diff-f055c70dd99867f8264dfbeb8f7b54b7355be4c3dc36ae576ba06940fa316d12R172), and "because I can".  But bringing into sync also will make the default value in auto-docs less confusing (actually correct).